source/main.lua

```diff
@@ -50,16 +50,16 @@ enum = require "enum"
 ss = require "socketstuff"
 
 garrLanders = {}
-garrGround = {}				-- stores the y value for the ground so that garrGround[Lander.x] = a value from 0 -> gintScreenHeight
-garrObjects = {}			-- stores an object that needs to be drawn so that garrObjects[xvalue] = an object to be drawn on the ground
+garrGround = {}	 -- y values for the ground so that garrGround[Lander.x] = a value from 0 -> gintScreenHeight

```

Comment:
It might add some new lines but I almost always prefer comments to be over the line instead of in-line comments.